### PR TITLE
[release branch] added greater alias documentation

### PR DESCRIPTION
**Pull request by scopatz**
Same a #137 but to the release branch.

### CODE REVIEW

**Comment by scopatz** [comment]
Also you might want to merge this PR @gonuke or someone else.

File: source/arche/cycpp.rst [hunk 1]
```diff
@@ -144,6 +144,15 @@ contains a listing of all special keys and their meanings.
                  in using this variable, default 0.
     alias        The name of the state variable in the schema and input file.
                  If this is not present it defaults to the C++ variable name. 
+                 The alias may also be a nested list of strings that matches 
+                 the C++ template type. Each member of the hierarchy will 
+                 recieve the cooresponding alias.  For example, a
```
Comment (mbmcgarry) [inline source/arche/cycpp.rst]: cooresponding -> corresponding

**Comment by scopatz** [comment]
spelling has been checked.

**Comment by mbmcgarry** [comment]
This was also manually merged into source with 2586f43..eecb291
thanks @scopatz !

**Pull request merged**